app/static/js/iSpindel_graph_socketio.js

```diff
@@ -0,0 +1,75 @@
+Highcharts.setOptions({
+  time: {
+      useUTC: false
+  }
+});
+
+Highcharts.chart(graph_data.chart_id, {
+chart: {
+  events: {
+    load: function () {
+      var self = this
+      var event_name = 'iSpindel_session_update|' + graph_data.chart_id
+      var socket = io.connect('//' + document.domain + ':' + location.port)
```

Comment:
Can you diff the javascript & html templates w/ the other ones?  I know I definitely moved out the io.connect to be in index.html (was causing load issues.. it shouldn't be in the js (not sure what else I changed)

Reply:
Good catch. Update javascript on iSpindel_graph_socketio.js and verfied index.html has the change.